src/devices/ledvance.ts

```diff
@@ -38,6 +44,38 @@ const definitions: Definition[] = [
         description: 'Panel TW LED 625 UGR19',
         extend: [ledvanceLight({colorTemp: {range: [153, 370]}, color: true})],
     },
+    {
+        zigbeeModel: ['PLUG EU EM T'],
+        model: '4099854295232',
+        vendor: 'LEDVANCE',
+        description: 'SMART+ Indoor Plug EU with Energy meter ',
+        fromZigbee: [fz.on_off, fz.metering],
```

Comment:
Please remove, configure fromZigbee, toZigbee and use: `extend: [onOff(), electricityMeter()],`